<?xml version="1.0" encoding="UTF-8"?>
<!--

(c) 2018 Ian Clark

This file is part of cuteboard

Cuteboard is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

Cuteboard is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with Cuteboard.  If not, see <http://www.gnu.org/licenses/>.

-->
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cuteboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="localOnly">
     <property name="text">
      <string>Connect to a remote sharing server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="user"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Passphrase:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="host">
       <property name="placeholderText">
        <string>example.com or example.com:12345</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Use the same user and passphrase on multiple machines to share clipboards between them. (The passphrase is stored in plaintext!)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>user</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>user</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>pass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localOnly</sender>
   <signal>clicked(bool)</signal>
   <receiver>host</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
